resources/views/noticia/visualizar.blade.php

```diff
@@ -1,7 +1,17 @@
+@guest
 <x-guest-layout>
     @component('layouts.nav_bar')@endcomponent
     <div class="container" style="margin-top: 50px; margin-bottom: 50px;">
         <div class="row justify-content-center">
+            <div class="col-md-10">
+                <div class="form-row">
+                    <div class="col-md-8">
+                    </div>
+                    <div class="col-md-4" style="text-align: right; padding-top: 15px;">
+                        <a title="Voltar" href="javascript:window.history.back();" ><img class="icon-licenciamento btn-voltar" src="{{asset('img/back-svgrepo-com.svg')}}" alt="Icone de voltar"></a>
```

Comment:
se não existir historico o botão se torna inútil, por exemplo no caso da noticia na página inicial que está abrindo em uma nova aba